### ISSUE TITLE
Annotations does not work with the theme feature navigation.instant

### ISSUE THREAD

**Issue by superhize** [squidfunk/mkdocs-material#3685]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [x] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

Annotations (https://squidfunk.github.io/mkdocs-material/reference/annotations/) does not work when the theme feature `navigation.instant` is enabled.

### Expected behaviour

Animations should work as it does in the officiel docs: https://squidfunk.github.io/mkdocs-material/reference/annotations/

### Actual behaviour

Annotations does not close as shown in the gif.
![](https://i.imgur.com/n8qeOI0.gif)

### Steps to reproduce

1. Use a clean mkdocs project
2. Use the material theme with insiders
3. Enable the theme feature `navigation.instant`
4. Enable the markdown extensions `pymdownx.superfences`, `attr_list` and `md_in_html`

### Package versions

- Python 3.10.2
- MkDocs 1.2.3
- Material 8.2.5+insiders.4.11.0

### Configuration

```yaml
site_name: Irezia - Wiki
repo_name: Irezia-Wiki on GitHub
copyright: "© 2022  Irezia"
docs_dir: docs
theme:
  name: material
  features:
    - navigation.instant
markdown_extensions:
  - pymdownx.superfences
  - attr_list
  - md_in_html
```

### System information

- Operating system: Windows 10
- Browser: Opera GX LVL3 (core: 83.0.4254.70), Firefox 97.0.1

**Comment by squidfunk** [comment]
Thanks for reporting! Fixed in [`72975c7`](https://github.com/squidfunk/mkdocs-material-insiders/commit/72975c708a72b9d6fa59e0d5f03d7221218bab10). There was indeed a side effect for annotations when instant loading was enabled. Sorry this took so long – I was on vacation in the meantime 😉

**Comment by squidfunk** [comment]
Released as part of 8.2.8+insiders-4.12.0.
